katas/langtonant/Giovanni/langtonsant.js

```diff
@@ -0,0 +1,61 @@
+const mapdim = 6;
+let map = new Array(mapdim).fill().map(() => Array(mapdim).fill(0));
+let visualmap = map;
+const ant = {};
+
+const LEFT = 3;
+const RIGHT = 1;
+
+ant.symbol = 8;
+ant.x = mapdim / 2;
+ant.y = mapdim / 2;
+ant.dir = 0;
+
+function turn(nextdir) {
+    ant.dir = (ant.dir + nextdir) % 4;
+}
+
+function advance() {
+    switch (ant.dir) {
+    case 0:
```

Comment:
Instead of 0 1 2 3, try to use constants so the code is more readable, for example:
```
const UP = 2
...
case UP:
```